### PR TITLE
add quizzes to DFS and Flood Fill sections

### PR DESCRIPTION
**Pull request by xyzyzl**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

fixed the big commit issue lol

### CODE REVIEW

File: content/3_Silver/DFS.mdx [hunk 1]
```diff
@@ -585,3 +585,40 @@ public class BuildingTeams
 ### Problems
 
 <Problems problems="bip" hideSuggestProblemButton />
+
+## Quiz
+Other questions concerning DFS and trees can be found in the Introduction to Tree Algorithms section.
```
Comment (nwatx) [inline content/3_Silver/DFS.mdx:590]: link the section

File: content/3_Silver/DFS.mdx [hunk 2]
```diff
@@ -585,3 +585,40 @@ public class BuildingTeams
...
+
+<Quiz>
+	<Quiz.Question>
+		Can a DFS with adjacency list be used to find shortest paths in linear time?
+		<Quiz.Answer>
+			Yes
+			<Quiz.Explanation>
+				Incorrect.
```
Comment (nwatx) [inline content/3_Silver/DFS.mdx:598]: I feel like the correct explanation could be pasted here.

Comment (nwatx) [reply content/3_Silver/DFS.mdx:598]: (from the correct answer)

File: content/3_Silver/DFS.mdx [hunk 3]
```diff
@@ -585,3 +585,40 @@ public class BuildingTeams
...
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			No
+			<Quiz.Explanation>
+				Correct! Worst case, a DFS shortest path algorithm would have to iterate through <i>every</i> path before finding the shortest. Working shortest path algorithms will be covered later in the Gold section.
```
Comment (nwatx) [inline content/3_Silver/DFS.mdx:604]: link to gold

**Review by nwatx — CHANGES_REQUESTED** [review]

File: content/3_Silver/DFS.mdx (hunk 2 above)
Comment (xyzyzl) [reply content/3_Silver/DFS.mdx:598]: benq told me not to

**Comment by xyzyzl** [comment]
it happened again????

**Comment by xyzyzl** [comment]
it literally deleted all of the above files and then re-created them. this is weird

**Comment by nwatx** [comment]
try again or fix :p

**Pull request closed without merging**